eval_protocol/pytest/default_single_turn_rollout_process.py

```diff
@@ -14,6 +15,15 @@ async def default_single_turn_rollout_processor(
 ) -> List[EvaluationRow]:
     """Generate a single response from any supported model provider using LiteLLM."""
 
+    # Explicitly disable LiteLLM caching to avoid reused responses across runs
+    try:
+        litellm.cache = None
+        # Some versions expose a helper; ignore if unavailable
+        if hasattr(litellm, "disable_cache"):
+            litellm.disable_cache()  # type: ignore[call-arg]
+    except Exception:
```

Comment:
Instead of doing this we should just be doing `cache={"no-cache": True}` as part of acompletion. i.e. should be something like:

```suggestion
request_params["cache"] = {"no-cache": True}
```